di_website/templates/events/event_index_page.html

```diff
@@ -0,0 +1,67 @@
+{% extends "layouts/default.html" %}
+
+{% load wagtailcore_tags %}
+
+{% block hero %}
+  {% include 'includes/heroes/hero.html' %}
+{% endblock hero %}
+
+{% block body %}
+
+<section class="section">
+  <div class="row row--narrow">
+      <div class="l-sidebar">
+          <aside class="l-sidebar__aside">
+              <div class="aside aside--clipping">
+                  
+<h2 class="aside__heading">More in: What we do</h2>
```

Comment:
Pattern Library templates are filled with placeholder text. It's meant as a guide on what we should expect the content to look like. Never copy it into your code. It's supposed to be dynamic, created and populate from the admin side. For this section, just leave a TODO comment that we'll add what we do when we build that page.

Reply:
I left the template code as is, and added comment as advised.